src/components/BreadcrumbsItem/BreadcrumbsItem.module.css

```diff
@@ -0,0 +1,50 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # BREADCRUMBS ITEM
+\*------------------------------------*/
+
+/**
+ * Breadcrumbs list item
+ */
+.breadcrumbs__item {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+/**
+ * Breadcrumbs link
+ */
+.breadcrumbs__link {
+  color: var(--eds-theme-color-text-neutral-subtle);
+  text-decoration: none;
```

Comment:
I'm super not a fan of this.. It's literally impossible to tell which of the bread crumbs are links without interacting with the text.

Reply:
I brought this up with design but apparently doesn't need to be the purple link color due to the context of its usage. I'll swing it by design again...

Reply:
Looking at it in the product and the examples in EDS, at least it looks like they are all always links (I've seen it before where the one furthest right isn't a link because it's the current page). I'm still not happy about it because it's perfectly possible for bread crumbs to be purely informative and not interactive.. but I dunno how hard we should push back on it because maybe all users do expect bread crumbs to be interactive; I don't know.